src/components/ebay-inline-notice/index.marko

```diff
@@ -8,4 +8,5 @@ $ var status = input.status || "attention";
     root="div"
     header-root="span"
     main-root="span"
+    a11y-text=input.a11yText
```

Comment:
This is not needed, `a11yText` should get passed through from the `...input`. 
If it's not, then we need to add `a11y-text` in marko.json
Check this again